chernoAlieu.Habit-Tracker/Habit-Tracker/Program.cs

```diff
@@ -0,0 +1,240 @@
+﻿using System.Data;
+using Microsoft.Data.Sqlite;
+using Microsoft.VisualBasic.FileIO;
+using System.Collections.Generic;
+using System.Globalization;
+
+
+class Program
+{
+	static string connectionString = @"Data Source=habit-Tracker.db";
+
+	static void Main(string[] args)
+	{
+		using (var connection = new SqliteConnection(connectionString))
+		{
+			connection.Open();
+			var tableCmd = connection.CreateCommand();
+
+			tableCmd.CommandText = @"CREATE TABLE IF NOT EXISTS drinking_water (
+								Id INTEGER PRIMARY KEY AUTOINCREMENT,
+								Date TEXT,
+								Quantity INTEGER
+								)";
+			tableCmd.ExecuteNonQuery();
+			connection.Close();
+		}
+
+		GetUserInput();
+
+	}
+
+	static void GetUserInput()
+	{
+		Console.Clear();
+		var closeApp = false;
+		while (closeApp == false)
+		{
+			Console.WriteLine("\n\nMAIN MENU");
+			Console.WriteLine("\nWhat would you like to do?");
+			Console.WriteLine("\nType 0 to Close Application.");
+			Console.WriteLine("Type 1 to View all Records");
+			Console.WriteLine("Type 2 to Insert Record");
+			Console.WriteLine("Type 3 to Delete Record");
+			Console.WriteLine("Type 4 to Update Record");
+			Console.WriteLine("-------------------------------------");
+
+			string commandInput = Console.ReadLine();
+
+			switch (commandInput)
+			{
+				case "0":
+					Console.WriteLine("\nGoodbye!\n");
+					closeApp = true;
+					Environment.Exit(0);
+					break;
+				case "1":
+					GetAllRecords();
+					break;
+				case "2":
+					Insert();
+					break;
+				case "3":
+					Delete();
+					break;
+				case "4":
+					Update();
+					break;
+				default:
+					Console.WriteLine("\nInvalid Command. Please type a number from 0 to 4.\n");
+					break;
+			}
+		}
+
+
+	}
+
+	private static void Insert()
+	{
+		string date = GetDateInput();
+
+		int quantity =
+			GetNumberInput("\n\nPlease insert number of glasses or other measure of your choice " +
+						   "(no decimals allowed)\n\n");
+
+		using (var connection = new SqliteConnection(connectionString))
+		{
+			connection.Open();
+			var tableCmd = connection.CreateCommand();
+			tableCmd.CommandText = $"INSERT INTO drinking_water(date, quantity) VALUES('{date}',{quantity})";
+
+			tableCmd.ExecuteNonQuery();
+			connection.Close();
+		}
+
+	}
+
+	internal static string GetDateInput()
+	{
+		Console.WriteLine("\n\nPlease insert the date: (Format: dd-mm-yyyy). Type 0 to return to main menu.");
```

Comment:
Looks like you are telling users to insert a year with 4 digits but your datetime parsing method is checking for year with 2 digits 🤓😁